atorch/examples/llama2/README.md

```diff
@@ -1,4 +1,48 @@
-## Intro
+# Llama2 Pretrain/Finetune
+
+This document presents examples of using ATorch to pretrain or finetune the HuggingFace Llama2 model, including FSDP (ZeRO3) and 3D hybrid parallelism approaches.
+
```

Comment:
Llama2 model and wikitext dataset is used in the examples. If you have already downloaded llama2 model, set environment variable:  xxx=llama2_model_directory
If you have wikitext in your system, set environment variable:  xxx=wikitext_data_directory

If llama2 model or wikitext dataset is not present, the training script will automatically download them for you. Note that downloading may take quite some time.